shared/src/TwinJsonConverter.cs

```diff
@@ -39,7 +39,7 @@ public sealed class TwinJsonConverter : JsonConverter
         private const string CloudToDeviceMessageCountTag = "cloudToDeviceMessageCount";
         private const string AuthenticationTypeTag = "authenticationType";
         private const string X509ThumbprintTag = "x509Thumbprint";
-        private const string ModelId = "digital-twin-model-id";
+        private const string ModelId = "modelId";
```

Comment:
so this is `modelId` and not `model-id`?

Reply:
yes

Reply:
https://github.com/Azure/azure-iot-sdk-java/pull/835/files as per this it model-id. So is it different between device client and service client?

Reply:
Okay this resolves it. https://github.com/Azure/azure-iot-sdk-java/pull/834/files